src/modules/apps/e2e/report.tsx

```diff
@@ -0,0 +1,270 @@
+import * as React from 'react'
+import { Box, Color, Text, Static } from 'ink'
+import { all, difference, groupBy, toPairs, values, countBy, sum, pathOr, path } from 'ramda'
+import { parseAppId } from '@vtex/api'
+
+import { SpecReport, AppReport, TestReport, SpecTestReport, Screenshot } from '../../../clients/Tester'
+import { useInterval } from './useInterval'
+
+type AppIdProps = {
+  appId: string
+}
+
+const AppId: React.FunctionComponent<AppIdProps> = ({ appId }) => {
+  const { name, version, build } = parseAppId(appId)
+  const [vendor, app] = name.split('.')
+  return (
+    <Box>
+      <Color blue>{vendor}</Color>
+      <Color dim>.</Color>
+      {`${app}@${version}`}
+      {build && <Color dim>{`-${build}`}</Color>}
+    </Box>
+  )
+}
+
+type SpecDetailProps = {
+  label: string
+  text: string
+  indented: boolean
+}
+
+const FailedSpecDetail: React.FunctionComponent<SpecDetailProps> = ({ label, text, indented }) => {
+  const nLines = text.split('\n')
+  return (
+    <Box flexDirection="column" marginBottom={1}>
+      <Text>
+        <Text underline>{`${label}:`}</Text>
+        {!indented && ` ${text}`}
+      </Text>
+      {indented && (
+        <Box marginTop={1}>
+          <Box flexDirection="column">{nLines.map(() => '│ ')}</Box>
+          <Box>{text}</Box>
+        </Box>
+      )}
+    </Box>
+  )
+}
+
+type SpecProps = {
```

Comment:
Prefer `interface`, it's the preferred way in toolbelt
```suggestion
interface SpecProps {
```